diff --git a/node_modules/ldk-node-rn/ios/LdkNodeRnModule.swift b/node_modules/ldk-node-rn/ios/LdkNodeRnModule.swift
index dd055a1..9ce5353 100644
--- a/node_modules/ldk-node-rn/ios/LdkNodeRnModule.swift
+++ b/node_modules/ldk-node-rn/ios/LdkNodeRnModule.swift
@@ -198,21 +198,6 @@ class LdkNodeRnModule: NSObject {
         }
     }
     
-    @objc
-    func setLiquiditySourceLsps2(_
-        address: String,
-        nodeId: String,
-        token: String,
-        buildId: String,
-        resolve: @escaping RCTPromiseResolveBlock,
-        reject: @escaping RCTPromiseRejectBlock
-    ) {
-        DispatchQueue.main.async { [self] in
-            _builders[buildId]!.setLiquiditySourceLsps2(address: address, nodeId: nodeId, token: token)
-            resolve(true)
-        }
-    }
-
     @objc
     func build(_
         builderId: String,
